plugins/module_utils/entity.py

```diff
@@ -411,8 +411,13 @@ def _fetch_url(
             return {"status_code": status_code}
 
         if resp_json is None:
+            if len(info.get("msg")) > 0:
```

Comment:
If `msg` is not present in `info` then TypeError will be raised due to `len(None)`. Is msg by default "" in info ?

Reply:
2. If `msg` attribute is present in `info` with value `None` still it will raise type error.

Reply:
I don't have access to a lab environment for a couple of days to check what an empty `msg` is, but I can add a conditional regarding `None`.

Reply:
Sure thanks !